### PR TITLE
feat: add bot as repo collaborator during import

### PR DESCRIPTION
**Pull request by mgoltzsche**
as the title says

### CODE REVIEW

**Comment by mgoltzsche** [comment]
unfortunately this PR is still not ready: it made the bot accept the invitation at some point. However after some cleaning up now it does not anymore because apparently it is listing invitations for my personal user instead of the bot somehow.

File: pkg/cmd/importcmd/import.go [hunk 1]
```diff
@@ -707,68 +712,66 @@ func (options *ImportOptions) CreateNewRemoteRepository() error {
...
 		// If the user creating the repo is not the pipeline user, add the pipeline user as a contributor to the repo
 		if options.PipelineUserName != options.GitUserAuth.Username && options.GitServer != nil && options.GitServer.URL == options.PipelineServer {
 			// Make the invitation
-			err := options.GitProvider.AddCollaborator(options.PipelineUserName, details.Organisation, details.RepoName)
+			err := options.GitProvider.AddCollaborator(options.PipelineUserName, options.Organisation, options.AppName)
+			if err != nil {
+				return err
+			}
+
+			authConfigSvc, err := options.GitAuthConfigService()
```
Comment (mgoltzsche) [inline pkg/cmd/importcmd/import.go:734]: What tricked me here: Previously `options.GitLocalAuthConfigService()` was used (for quickstart) which used the local git credentials. In order to retrieve the pipeline bot's credentials from vault `options.GitAuthConfigService()` must be used and the credentials obtained using `authConfigSvc.Config().GetPipelineAuth()`.

**Comment by romainverduci** [comment]
This looks good to me and is actually working when I'm testing it locally with both the UI and the CLI.
Would be nice to have a review from someone with more knowledge in this area.
@abayer @hferentschik maybe ?

**Pull request merged**